Lib/test/pythoninfo.py

```diff
@@ -376,6 +376,9 @@ def collect_gdb(info_add):
                                 stderr=subprocess.PIPE,
                                 universal_newlines=True)
         version = proc.communicate()[0]
+        if proc.returncode:
```

Comment:
I consider this more readable:

```suggestion
        if proc.returncode != 0:
```

Reply:
Both syntaxes are equivalent. I prefer `if proc.returncode:" (coding style preference) ;-)